CONTRIBUTING.md

```diff
@@ -35,7 +35,7 @@ make test
 
 ### Running local build with the UI
 
-The `jaeger-ui` submodule contains the source code for the UI assets (requires Node.js 6+). The assets must be compiled first with `make build-ui`, which runs Node.js build and then packages the assets into a Go file that is `.gitignore`-ed. The packaged assets can be enabled by providing a build tag `ui`, e.g.:
+The `jaeger-ui` submodule contains the source code for the UI assets (requires Node.js 6+). The assets must be compiled first with `make build_ui`, which runs Node.js build and then packages the assets into a Go file that is `.gitignore`-ed. The packaged assets can be enabled by providing a build tag `ui`, e.g.:
```

Comment:
The instructions are correct as is, the target name is `build-ui`:

https://github.com/jaegertracing/jaeger/blob/2957d89a58814c43c20f7fc10c737ebef95460ea/Makefile#L195-L199